models.py

```diff
@@ -194,7 +194,7 @@ def __init__(self, controller, **kwargs):
         super().__init__()
 
         # required attributes
-        self._id = kwargs.get('id', uuid4().hex[:16])
+        self._id = kwargs.get('id', uuid4().hex)[:14]
```

Comment:
This is silently ignoring the first two bytes passed as arguments, right? I think this should be explicitly documented somewhere.

Reply:
It is not ignoring the first two bytes passed, it is using only the first 14 bytes and ignoring anything else.
I've added an explanation in the method's docstring.